ir_disasm.c

```diff
@@ -365,7 +365,7 @@ int ir_disasm(const char    *name,
 	}
 #  endif
 	cs_option(cs, CS_OPT_DETAIL, CS_OPT_ON);
-#  if DISASM_INTEL_SYNTAX
+#  ifdef DISASM_INTEL_SYNTAX
```

Comment:
It seems this code was copy-pasted from the old PHP JIT.
I think, It's better to define IR_DISASM_INTEL_SYNTAX at start.

```c
#ifned IR_DISASM_INTEL_SYNTAX
# define IR_DISASM_INTEL_SYNTAX 0
#endif
```